extras/manifests/prometheus-rules.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
```

Comment:
name of Prometheus is alert https://github.com/gluster/gcs/blob/29531c63d6c0e602dffee0a744084b2562da536b/deploy/templates/gcs-manifests/gcs-prometheus-bundle.yml.j2#L79

Reply:
Ack.
But this yaml contains alerts and recording rules both. Naming it something like `alert` or `rules` is just misleading and makes no sense. 
Instead we should name it `gluster` or something similar. 

This change would have to be made in both `gcs` and `gluster-mixins`. Thoughts?
@cloudbehl @shtripat